src/RequestHandler/outofprocess/serverprocess.cxx

```diff
@@ -747,6 +747,7 @@ SERVER_PROCESS::StartProcess(
 
     while (dwRetryCount > 0)
     {
+        m_dwPort = 0;
```

Comment:
Put this in CleanUp?

Reply:
Its better to put here as if the failure happens earlier, we don't want to log an initialized value